src/lib/baseCommand.ts

```diff
@@ -86,13 +88,23 @@ export default class Command {
    */
   args: OptionDefinition[];
 
-  run(opts: CommandOptions<{}>): Promise<string> {
+  async run(opts: CommandOptions<{}>): Promise<string> {
     Command.debug(`command: ${this.command}`);
     Command.debug(`opts: ${JSON.stringify(opts)}`);
 
     if (this.args.some(arg => arg.name === 'key')) {
       if (!opts.key) {
-        throw new Error('No project API key provided. Please use `--key`.');
+        if (isCI()) {
+          throw new Error('No project API key provided. Please use `--key`.');
+        } else {
```

Comment:
Probably don't need an `else` here and I'm surprised ESLint isn't complaining about it.

Reply:
Oof, nice catch thank you! See https://github.com/readmeio/rdme/pull/636/commits/ce6455da39062952fa55cd86ab4d58221015f894